cmake/Modules/OpmCompile.cmake

```diff
@@ -13,11 +13,13 @@ macro (opm_compile opm)
   # all public header files are together with the source. prepend our own
   # source path to the one of the dependencies so that our version of any
   # ambigious paths are used.
-  set (${opm}_INCLUDE_DIR "${PROJECT_SOURCE_DIR}")
-  set (${opm}_INCLUDE_DIRS ${${opm}_INCLUDE_DIR} ${${opm}_INCLUDE_DIRS})
+  # set (${opm}_INCLUDE_DIR "${PROJECT_SOURCE_DIR}")
+  # set (${opm}_INCLUDE_DIRS ${${opm}_INCLUDE_DIR} ${${opm}_INCLUDE_DIRS})
 
   # create this library, if there are any compilation units
-  include_directories (${${opm}_INCLUDE_DIRS})
+  # include_directories (${${opm}_INCLUDE_DIRS})
+  include_directories("${PROJECT_SOURCE_DIR}")
+  include_directories (SYSTEM ${${opm}_INCLUDE_DIRS})
```

Comment:
I'm not really sure what exactly causes the issue which I mentioned on #207. I just know that the `-I` flags for some modules suddenly disappeared and that locally reverting #202 fixed it. I consider it very possible that my issue is some weird side effect, though...